runst.sh

```diff
@@ -0,0 +1,17 @@
+export LD_LIBRARY_PATH="$(pwd)/third_party/motr/motr/.libs/:"\
```

Comment:
whats the need of this script

Reply:
Just a way to run STs. Got commited at some point.

Removed from this patch.